openedx/core/djangoapps/django_comment_common/comment_client/models.py

```diff
@@ -101,25 +101,6 @@ def _metric_tags(self):
     def find(cls, id):  # pylint: disable=redefined-builtin
         return cls(id=id)
 
-    @classmethod
-    def retrieve_all(cls, params=None):
-        """
-        Performs a GET request against the resource's listing endpoint.
-
-        Arguments:
-            params: A dictionary of parameters to be passed as the request's query string.
-
-        Returns:
-            The parsed JSON response from the backend.
-        """
-        return perform_request(
```

Comment:
Shouldn't this become a call to something in the forums_api, rather than get removed entirely? Isn't it still being called from:

https://github.com/edly-io/edx-platform/blob/3679a7abdbf773cf7f98ad60c0bb3d40b5f41bad/lms/djangoapps/discussion/rest_api/api.py#L1778-L1785

?

Reply:
Ah, got it, thank you.